11ty/definitions/semitic.md

```diff
@@ -1,5 +1,23 @@
 ---
 title: Semitic
 slug: semitic
-defined: false
----
\ No newline at end of file
+flag: _ _
+defined: true
+excerpt: 
+    1. Related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa. There are included Arabic, Hebrew, Aramaic and Amharic.
+    2. Mainly referred to as Arabic and Hebrew
+speech: adjective
+alt_words:
+- _ _
+- _ _
```

Comment:
```suggestion
```

Since we're not offering alternate words here (an optional property), we can erase all this.